docs/features/logforwarding/outputs/syslog-forwarding.adoc

```diff
@@ -0,0 +1,149 @@
+  The value can be a decimal integer. Facility keywords are not standardized,
+  this API recognizes at least the following case-insensitive keywords
+  (defined by https://en.wikipedia.org/wiki/Syslog#Facility_Levels):
+
+    kernel user mail daemon auth syslog lpr news
+    uucp cron authpriv ftp ntp security console solaris-cron
+    local0 local1 local2 local3 local4 local5 local6 local7
+
+
+. `payloadKey`: specifies record field to use as payload. This supports template syntax to allow dynamic per-event values.
+  The `payloadKey` must be a single field path encased in single curly brackets `{}`.
+  Field paths must only contain alphanumeric and underscores. Any field with other characters must be quoted.
+  If left empty, Syslog will use the whole message as the payload key.
+  Examples:
+
+  {.bar}
+  {.foo.bar.baz}
+  {.foo.bar."baz/with/slashes"}
+
+. `appName`: is APP-NAME part of the syslog-msg header.
+  `appName` needs to be specified if using `rfc5424`. The maximum length of the final values is truncated to 48. This supports template syntax* to allow dynamic per-event values.
+
+.  `procId`: is `PROCID` part of the syslog-msg header.
+   `procId` needs to be specified if using `rfc5424`. The maximum length of the final values is truncated to 128. This supports template syntax* to allow dynamic per-event values.
+
+.  `msgId`: is `MSGID` part of the syslog-msg header. This supports template syntax to allow dynamic per-event values. MsgId needs to be specified if using `rfc5424`.  The maximum length of the final values is truncated to 32. This supports template syntax* to allow dynamic per-event values.
+
+. `enrichment`: `None` or `KubernetesMinimal`.
+`None` add no additional enrichment to the record.
+`KubernetesMinimal` adds `namespace_name`, `pod_name`, and `container_name` to the beginning of the message
+body (e.g. `+namespace_name=myproject, container_name=server, pod_name=pod-123, message={"foo":"bar"}+`).
+This may result in the message body being an invalid JSON structure.
+
+
+[NOTE]
+====
+*Template syntax
+
+The `appName`, `msgId` and `procId` can be a combination of static and dynamic values consisting of field paths followed by `||` followed by another field path or a static value.
+A dynamic value is encased in single curly brackets `{}` and MUST end with a static fallback value separated with `||`.
+Static values can only contain alphanumeric characters along with dashes, underscores, dots and forward slashes.
+
+Examples:
+
+  foo-{.bar||"none"}
+  {.foo||.bar||"missing"}
+  foo.{.bar.baz||.qux.quux.corge||.grault||"nil"}-waldo.fred{.plugh||"none"}
+====
+
+=== Defaults value of Syslog fields
+Will be calculating during processing log event if values not set:
+
+==== RFC3164
+
+Format: `+<PRI>TIMESTAMP HOSTNAME TAG: MESSAGE+`
+
+Example: `+<34>Oct 11 22:14:15 mymachine su[1234]: 'su root' failed for lonvick on /dev/pts/+`
+
+|===
+| | journal |application|infrastructure|note
```

Comment:
We have a little bit of conflicting statements here because of how we define log type.  Infra is container or journal where application is only container.  We should probably change app and infra to just be "container" and then add "audit" as another header since we process those.  Also the header could be something like:

Infrastructure Journal, Infrastructure/Application Container, Audit